fea-rs/src/compile/compile_ctx.rs

```diff
@@ -1888,6 +1897,37 @@ impl<'a, F: FeatureProvider, V: VariationInfo> CompilationCtx<'a, F, V> {
             }
         } else if item.kind() == Kind::Semi {
             // continue
+        } else if item.kind() == Kind::Comment {
+            assert!(item
+                .token_text()
+                .unwrap_or_default()
+                .contains("# Automatic Code"));
+            if self.active_feature.is_none() {
+                self.warning(
+                    item.range(),
+                    "Insertion marker outside feature block will be ignored",
+                );
+                return;
+            };
+
+            // make sure we finish any active lookup before assigning the lookupid
+            if let Some((id, _name)) = self.lookups.finish_current() {
+                assert!(
+                    _name.is_none(),
+                    "named lookup should be finished before insert marker"
+                );
```

Comment:
If this is a bad state lets error. If it's not lets warn. Either way lets not assert.

Reply:
I was thinking this is an invariant but actually this could occur in the wild if someone wrote,

```fea
feature test {
  lookup oops {
    # Automatic Code
  } oops;
} test;
```

that is, they put the comment inside a lookup block inside a feature.

Updated to just error in this case.